lib/manageiq/api/common/graphql/generator.rb

```diff
@@ -100,7 +103,7 @@ def self.init_schema(request, schema_overlay = {})
             resources.each do |resource|
               next unless openapi_content.dig("paths", resource, "get") # we only care for queries
 
-              rmatch = resource.match("^/(.*/)?([^/]*)/{id}$")
+              rmatch = resource.match("^/(.*/)?([^/]*)/{[[a-z]*_]*id}$")
```

Comment:
Can this be `/^/(.*/)?([^/]*)/{[a-z_]*id}$/`

Reply:
same in the other examples

Reply:
I had that for a bit, but it allows for prefixes with no _, so attrid vs requiring attr_id. I wanted either {id} or {..._id}.  wdyt ?